main.tf

```diff
@@ -79,6 +79,16 @@ resource "aws_security_group_rule" "egress_service" {
   ipv6_cidr_blocks  = ["::/0"]
 }
 
+resource "aws_security_group_rule" "ingress_service" {
+  security_group_id = aws_security_group.ecs_service.id
+  type              = "ingress"
+  protocol          = "tcp"
+  from_port         = 0
+  to_port           = var.task_container_port
+  cidr_blocks       = [var.vpc_cidr_block]
+  ipv6_cidr_blocks  = ["::/0"]
```

Comment:
Make this one a variable as well, so we don't accidentally open for all IPv6 adresses.